src/python/pants/engine/rules_test.py

```diff
@@ -471,79 +476,6 @@ async def a_from_b(b: B) -> A:
             "Query or Get."
         ) in str(cm.value)
 
-    @pytest.mark.skip(reason="TODO(#10649): figure out if this tests is still relevant.")
```

Comment:
See https://github.com/pantsbuild/pants/issues/10649 for why we can kill these tests (which in any case we weren't running...)